resolutions/-resolution-xxx.md

```diff
@@ -0,0 +1,22 @@
+﻿# Hypha Co-op: Resolution Template
+
+<!-- Title: -->
+# [ Ordinary / Special ] resolution [action]
+<!-- Ex: Resolution amending section 18 (Quorum) of the Bylaws -->
+
+<sup>[From template][template]</sup>
+
+<!-- Preamble: -->
+Whereas...
+<!-- a statement about the need for the resolution. -->
+<!-- Ex: the co-operative wants to ensure quorum is achieved at members’ meetings -->
+
+<!-- Resolution: -->
+Therefore be it resolved that...
+<!-- the decision or statement -->
+
+Signed by President: [president's name]
+Date: YYYY-MM-DD
+
+<!-- Links -->
+   [template]: -resolution-xxx.md
```

Comment:
Can we remove this in the filename? There is no `-` in the actual file! We only do this is we decide to put `YYYY-MM-DD` ahead of it which may not be a bad idea. Though we should probably include `YYYY-MM-DD` as part of the filename. Leading with `-` makes all the cli editors think it's a parameter.

```suggestion
   [template]: resolution-xxx.md
```

Reply:
ah meant to prefix with the dash in filename as well. I'm fine either way. Thought dash was an informal convention for keeping templates at top on github

Reply:
date prefix is fine by me too. though numbers should keep things in order as well, unless i'm missing some consideration